notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-761_vms_io_and_plotting.ipynb

```diff
@@ -0,0 +1,436 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.    def vms_data_to_frame(filename, vms_type, begin_time=None, end_time=None):
</pre><p>Suggest changing name to vms_data_to_dataframe or vms_data_to_pandas to be more explicit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>